app/models/course.rb

```diff
@@ -11,6 +11,8 @@ class Course < ApplicationRecord
 
   delegate :short_title, to: :path, prefix: true
 
+  scope :badges, -> { Course.where(show_on_homepage: true) }
```

Comment:
A scope here is probably overkill, but I felt it helped document the purpose of the attribute a little better.

Reply:
I think having a scope for this is perfect, definitely not overkill mate :)